doc/source/user_guide/io.rst

```diff
@@ -169,7 +169,7 @@ dtype_backend : {"numpy_nullable", "pyarrow"}, defaults to NumPy backed DataFram
   implementation when "numpy_nullable" is set, pyarrow is used for all
   dtypes if "pyarrow" is set.
 
-  The dtype_backends are still experimential.
+  The dtype_backends are still experiential.
```

Comment:
should be "experimental"?

Reply:
https://github.com/pandas-dev/pandas/pull/59669